packages/sanity/src/structure/panes/document/documentPanel/header/DocumentPanelHeader.tsx

```diff
@@ -138,7 +140,7 @@ export const DocumentPanelHeader = memo(
           border
           ref={ref}
           loading={connectionState === 'connecting'}
-          title={<DocumentHeaderTitle />}
+          title={documentPaneHeaderTitle}
```

Comment:
Do we need this change? I don't think it's incorrect, but I also think if we could changes to the minimum in files that are not strictly related to `corel` will make it easier to review the `corel` branch down the road.

Reply:
Had the same thought. In general we should try to avoid including non-relevant changes like this, because of the noise and porential for merge conflicts it adds. Unrelated changes that are worth doing (e.g. improving readability), should preferable be done as a separate PR into main/next.